magento/magento2
# 12110: Missing cascade into attribute set deletion.

### Description
Fix for: Missing cascade into attribute set deletion.

### Fixed Issues (if relevant)

1. magento/magento2#12110: Missing cascade into attribute set deletion

### Manual testing scenarios
1. Create a custom attributes set.
2. Attach some products to this attributes set.
3. Got to attributes set admin page.
4. Delete your custom attributes set.
5. Confirm you want to remove attached products also.
6. Check, table url_rewrite should be cleaned from references to removed products.
7. Check, that new product named like an old one from removed attribute set can be created via API or admin interface.

### Contribution checklist
